### PR TITLE
GITC-657 Update l2 text for multi-sig wallets

### PR DESCRIPTION
**Pull request by ksolo**
##### Description

Update warnings for L2 based contributions

##### Refers/Fixes

GITC-657

### CODE REVIEW

File: app/grants/templates/grants/detail/template-grant-details.html [hunk 1]
```diff
@@ -445,8 +445,8 @@ <h3>About</h3>
                 <i class="fal fa-wallet mr-1"></i>
                 You’ve received L2 contributions
               </p>
-              Some of your grant funds may be in L2 (zkSync or Polygon).
-              Read more on how to withdraw from L2 wallets.
+              Some of your grant funds may be in L2 (zkSync or Polygon). Polygon contribution is not supported
```
Comment (gdixon) [inline app/grants/templates/grants/detail/template-grant-details.html:448]:
```suggestion
              Some of your grant funds may be in L2 (zkSync or Polygon). Polygon contributions are not supported
```

**Comment by thelostone-mc** [comment]
Is this ready for review @ksolo ?

**Comment by ksolo** [comment]
@thelostone-mc It's not I had some questions about requirements and paused this while I jumped on the Cypress PR. The questions have been answered now and I can jump back to this.

File: app/app/context.py [hunk 2]
```diff
@@ -114,9 +114,9 @@ def preprocess(request):
         onboard_tasks = []
 
     # town square wall post max length
-    max_length_offset = abs(((
-        request.user.profile.created_on if user_is_authenticated_and_valid else timezone.now()
-    ) - timezone.now()).days)
+    max_length_offset = abs(
+        ((request.user.profile.created_on if user_is_authenticated_and_valid else timezone.now()) - timezone.now()).days
+    )
```
Comment (ksolo) [inline app/app/context.py:119]: result of make fix

File: app/app/sitemaps.py [hunk 3]
```diff
@@ -191,9 +191,7 @@ class ActivitySitemap(Sitemap):
     limit = 5000
 
     def items(self):
-        return Activity.objects.exclude(
-            activities_index__key__isnull=True
-        ).order_by('-pk').cache()
+        return Activity.objects.exclude(activities_index__key__isnull=True).order_by('-pk').cache()
```
Comment (ksolo) [inline app/app/sitemaps.py:194]: result of make fix

File: app/avatar/management/commands/fix_broken_avatars.py [hunk 4]
```diff
@@ -25,7 +25,6 @@ class Command(BaseCommand):
 
     help = 'fixes avatars whose pngs have failed'
 
-
```
Comment (ksolo) [inline app/avatar/management/commands/fix_broken_avatars.py:28]: result of make fix

File: app/avatar/views_3d.py [hunk 5]
```diff
@@ -771,11 +771,15 @@ def get_avatar_attrs(theme, key):
             'path': 'assets/v2/images/avatar3d/joker.svg',
         },
     }
-    new_avatars = ['SSShiine','avrilapril','eknobl','lkh','nanshulot','wahyu243','zak102','artipedia','azizyano','hasssan04','maystro4','riyasoganii','writeprovidence', 'petushka1', 'merit-tech', 'hamzaghz', 'bruno-tandon', 'KayZou', 'the-hack-god', 'old-monger', 'virtual-face', 'art-maniac', 'vinhbhn', 'masket-bask', 'panigale120']
+    new_avatars = [
+        'SSShiine', 'avrilapril', 'eknobl', 'lkh', 'nanshulot', 'wahyu243', 'zak102', 'artipedia', 'azizyano',
+        'hasssan04', 'maystro4', 'riyasoganii', 'writeprovidence', 'petushka1', 'merit-tech', 'hamzaghz',
+        'bruno-tandon', 'KayZou', 'the-hack-god', 'old-monger', 'virtual-face', 'art-maniac', 'vinhbhn', 'masket-bask',
+        'panigale120'
+    ]
```
Comment (ksolo) [inline app/avatar/views_3d.py:779]: make fix

File: app/avatar/views_3d.py [hunk 6]
```diff
@@ -771,11 +771,15 @@ def get_avatar_attrs(theme, key):
...
     for _key in new_avatars:
         avatar_attrs[_key] = {
-            'preview_viewbox': {
-            },
+            'preview_viewbox': {},
```
Comment (ksolo) [inline app/avatar/views_3d.py:782]: make fix

File: app/dataviz/d3_views.py [hunk 7]
```diff
@@ -111,6 +111,7 @@ def data_viz_helper_get_data_responses(request, visual_type):
 
     return data_dict
 
+
```
Comment (ksolo) [inline app/dataviz/d3_views.py:114]: make fix

File: app/git/admin.py [hunk 8]
```diff
@@ -18,8 +18,10 @@
 
 """
 from django.contrib import admin
+
```
Comment (ksolo) [inline app/git/admin.py:21]: make fix

File: app/git/admin.py [hunk 9]
```diff
@@ -18,8 +18,10 @@
...
 from git.models import GitCache
 
+
```
Comment (ksolo) [inline app/git/admin.py:24]: make fix

File: app/git/tests/factories/git_cache_factory.py [hunk 10]
```diff
@@ -16,4 +16,3 @@ class Meta:
 
     # Generate binary data depending on n
     data = factory.Sequence(lambda n: ("{n}" * 100).encode("utf-8"))
-
```
Comment (ksolo) [inline app/git/tests/factories/git_cache_factory.py:19]: make fix

File: app/git/tests/models/test_git_cache.py [hunk 11]
```diff
@@ -1,6 +1,6 @@
-from git.tests.factories.git_cache_factory import GitCacheFactory
```
Comment (ksolo) [inline app/git/tests/models/test_git_cache.py:1]: make fix

File: app/git/tests/models/test_git_cache.py [hunk 12]
```diff
@@ -1,6 +1,6 @@
...
 import pytest
 from git.models import GitCache
+from git.tests.factories.git_cache_factory import GitCacheFactory
```
Comment (ksolo) [inline app/git/tests/models/test_git_cache.py:3]: make fix

File: app/git/tests/test_utils.py [hunk 13]
```diff
@@ -18,23 +18,23 @@
...
+from git.models import GitCache
+from git.tests.factories.git_cache_factory import GitCacheFactory
 from git.utils import (
-    HEADERS, TOKEN_URL, build_auth_dict, delete_issue_comment, get_github_emails, get_github_primary_email,
+    HEADERS, TOKEN_URL, _get_user, build_auth_dict, delete_issue_comment, get_github_emails, get_github_primary_email,
     get_issue_comments, get_issue_timeline_events, github_connect, is_github_token_valid, org_name, patch_issue_comment,
-    post_issue_comment, post_issue_comment_reaction, repo_url, reset_token, revoke_token, _get_user
+    post_issue_comment, post_issue_comment_reaction, repo_url, reset_token, revoke_token,
 )
+from github import NamedUser
```
Comment (ksolo) [inline app/git/tests/test_utils.py:37]: make fix

File: app/git/tests/test_utils.py [hunk 14]
```diff
@@ -18,23 +18,23 @@
...
 from unittest.mock import MagicMock, patch
-from github import NamedUser
+from urllib.parse import quote_plus, urlencode
 
 from django.conf import settings
 from django.test.utils import override_settings
 from django.utils import timezone
-from git.models import GitCache
-from git.tests.factories.git_cache_factory import GitCacheFactory
-from faker import Faker
```
Comment (ksolo) [inline app/git/tests/test_utils.py:30]: make fix

File: app/git/tests/test_utils.py [hunk 15]
```diff
@@ -289,4 +289,4 @@ def dump_mock(user_obj, file_obj):
         gh_client.get_user.assert_not_called()
         gh_user.update.assert_called_once()
 
-        assert loaded_user == gh_user
\ No newline at end of file
+        assert loaded_user == gh_user
```
Comment (ksolo) [inline app/git/tests/test_utils.py:292]: make fix

File: app/git/utils.py [hunk 16]
```diff
@@ -20,20 +20,21 @@
...
 import dateutil.parser
 import requests
-from .models import GitCache
 from github import Github
 from github.GithubException import BadCredentialsException, GithubException, UnknownObjectException
 from requests.exceptions import ConnectionError
 from rest_framework.reverse import reverse
 
+from .models import GitCache
+
```
Comment (ksolo) [inline app/git/utils.py:37]: make fix

File: app/grants/templates/grants/detail/template-grant-details.html [hunk 17]
```diff
@@ -649,7 +655,7 @@ <h5 class="font-weight-bold mb-3">Verify Grant Ownership with Twitter</h5>
           <label class="font-subheader letter-spacing text-grey-500 mb-0 mr-3" for="title">Logo background color</label>
           <input type="color" v-model="logoBackground" @input="changeColor">
         </div>
-        
+
```
Comment (ksolo) [inline app/grants/templates/grants/detail/template-grant-details.html:658]: make fix

File: app/marketing/utils.py [hunk 18]
```diff
@@ -28,7 +28,7 @@
 from django.utils.translation import gettext_lazy as _
 
 from mailchimp3 import MailChimp
-from marketing.models import AccountDeletionRequest, EmailSupressionList, LeaderboardRank, EmailSubscriber
+from marketing.models import AccountDeletionRequest, EmailSubscriber, EmailSupressionList, LeaderboardRank
```
Comment (ksolo) [inline app/marketing/utils.py:31]: make fix

File: app/marketing/views.py [hunk 19]
```diff
@@ -48,7 +48,7 @@
 from marketing.country_codes import COUNTRY_CODES, COUNTRY_NAMES, FLAG_API_LINK, FLAG_ERR_MSG, FLAG_SIZE, FLAG_STYLE
 from marketing.mails import new_feedback
 from marketing.models import AccountDeletionRequest, EmailSubscriber, Keyword, LeaderboardRank, UpcomingDate
-from marketing.utils import get_or_save_email_subscriber, validate_slack_integration, delete_email_subscription
+from marketing.utils import delete_email_subscription, get_or_save_email_subscriber, validate_slack_integration
```
Comment (ksolo) [inline app/marketing/views.py:51]: make fix

**Comment by ksolo** [comment]
I labeled the files that were updated from the `make fix` command so that reviewers can safely ignore them.

**Comment by nutrina** [comment]
There are quite a number of changes introduced by `make  fix`.
@ksolo @thelostone-mc @gdixon do you think it would make sense to use something like pre-commit: https://pre-commit.com

This would run a validation script on the changed files before every commit, and would ensure a common styling.
I must however admit, it can be annoying sometimes.

**Review by nutrina — APPROVED** [review]
Looks good to me.

**Comment by frankchen07** [comment]
LGTM

**Review by Apdlrcjafg19 — COMMENTED** [review]
Avance

**Pull request merged**